frontend/src/components/MeetingRoom/ToolbarOverflowMenu/ToolbarOverflowMenu.tsx

```diff
@@ -56,70 +64,81 @@ const ToolbarOverflowMenu = ({
+        key="ReportIssueButton"
+      />
+    ),
+    <ParticipantListButton
+      isOpen={rightPanelActiveTab === 'participant-list'}
+      handleClick={closeMenuWrapper(toggleParticipantList)}
+      participantCount={participantCount}
+      isOverflowButton
+      key="ParticipantListButton"
+    />,
+    <ChatButton
+      isOpen={rightPanelActiveTab === 'chat'}
+      handleClick={closeMenuWrapper(toggleChat)}
+      isOverflowButton
+      key="ChatButton"
+    />,
+  ];
+
   return (
-    <Popper open={isOpen} anchorEl={anchorRef.current} transition disablePortal placement="bottom">
-      {({ TransitionProps, placement }: PopperChildrenProps) => (
-        <Grow
-          {...TransitionProps}
-          style={{
-            transformOrigin: placement === 'bottom' ? 'center top' : 'center bottom',
-            width: 'calc(100dvw - 30px)',
-            left: '-15px',
-            position: 'relative',
-            translate: '0px -9px',
-          }}
-        >
-          <div className="flex w-full text-left font-normal">
-            <ClickAwayListener onClickAway={closeMenu}>
-              <Paper
-                data-testid="toolbar-overflow-menu"
-                className="flex items-center justify-center"
-                sx={{
-                  backgroundColor: '#272c2f',
-                  color: '#fff',
-                  padding: { xs: 1 },
-                  borderRadius: 2,
-                  zIndex: 1,
-                  width: '100%',
-                  position: 'relative',
-                  display: 'flex',
-                  justifyContent: 'center',
-                  alignContent: 'space-between',
-                  alignItems: 'center',
-                  paddingLeft: '12px',
-                }}
-              >
-                <LayoutButton
-                  isScreenSharePresent={isViewingScreenShare}
-                  isPinningPresent={isPinningPresent}
-                />
-                <EmojiGridButton
-                  isEmojiGridOpen={isEmojiGridOpen}
-                  setIsEmojiGridOpen={setIsEmojiGridOpen}
-                  isParentOpen={isOpen}
-                />
-                <ArchivingButton handleClick={closeMenu} />
-                {isReportIssueEnabled && (
-                  <ReportIssueButton
-                    isOpen={rightPanelActiveTab === 'issues'}
-                    handleClick={closeMenuWrapper(toggleReportIssue)}
-                  />
-                )}
-                <ParticipantListButton
-                  isOpen={rightPanelActiveTab === 'participant-list'}
-                  handleClick={closeMenuWrapper(toggleParticipantList)}
-                  participantCount={participantCount}
-                />
-                <ChatButton
-                  isOpen={rightPanelActiveTab === 'chat'}
-                  handleClick={closeMenuWrapper(toggleChat)}
-                />
-              </Paper>
-            </ClickAwayListener>
-          </div>
+    <Portal>
```

Comment:
just curious what are the benefits of using a `Portal` here instead of `Popper` that we had? to be fair implementation looks cleaner but I'm wondering if there are any other reasons beside that

Reply:
`Popper` is tied to an element, so its placement is tied to it as well. Using a `Portal` gives us flexibility in positioning.

Reply:
nice, thanks, we might consider doing _some refactoring_ for other popup but obviously not in this PR 🙏